src/views/demoDetails/HighlightBox.tsx

```diff
@@ -314,12 +315,15 @@ function PointCapturedHighlightBox(
     icon = "bluecapture";
   }
 
+  const translationLookup = highlight.point_name.substring(1); // substring(1) to remove the leading "#" character
+  const pointName = CapturePoints[translationLookup as keyof typeof CapturePoints] ?? highlight.point_name;
+
```

Comment:
I'm not sure all capture points have localized names. I think it's cleaner to check for `point_name.startsWith("#")` before looking up the name.